examples/api/test/material/navigation_rail/navigation_rail.extended_animation.0_test.dart

```diff
@@ -0,0 +1,65 @@
+// Copyright 2014 The Flutter Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+import 'package:flutter/material.dart';
+import 'package:flutter_api_samples/material/navigation_rail/navigation_rail.extended_animation.0.dart'
+    as example;
+import 'package:flutter_test/flutter_test.dart';
+
+void main() {
+  testWidgets('Navigation rail should animated itself between the normal and extended state',
```

Comment:
```suggestion
  testWidgets('Navigation rail animates itself between the normal and extended state',
```
Just a suggestion, apply it only if you think it is better.